packages/commonjs/src/index.js

```diff
@@ -156,24 +250,56 @@ export default function commonjs(options = {}) {
         });
       }
 
+      // TODO: For code splitting, the runtime probably needs to be imported by each entry point
```

Comment:
Probably? This is one of the two most important plugins of the ecosystem. Why not write a test and if it does not work, address this? This is important!

Reply:
@lukastaegert This comment is by you, after you did some refactoring on the original repo.

Could you help with a test for this one? I've never tried codesplitting in rollup, no experience with it. Don't want any mistakes there.